src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-thinc.py

```diff
@@ -0,0 +1,20 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the Apache License 2.0
+#
+# The full license is available in LICENSE.APL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: Apache-2.0
+# ------------------------------------------------------------------
+
+"""
+Thinc contains data files and hidden imports. This hook was created to make spacy work correctly.
+"""
+from PyInstaller.utils.hooks import collect_data_files, collect_submodules, requirements_for_package
+
+datas = collect_data_files("thinc")
+hiddenimports = collect_submodules("thinc")
+hiddenimports += requirements_for_package("thinc")
```

Comment:
Is this this `requirements_for_package()` really necessary?

Reply:
Good question. I can't figure out why I added that, everything seems to work ok without it. My best guess is that something I fixed in a different hook was killing my test runs when I built the thinc hook, and this line became redundant once I got the logic in the correct hook. I'll remove it for now and make sure nothing breaks.